config/locales/simple_form.en.yml

```diff
@@ -65,6 +65,7 @@ en:
         phrase: Keyword or phrase
         setting_auto_play_gif: Auto-play animated GIFs
         setting_boost_modal: Show confirmation dialog before boosting
+        setting_default_federation: Always allow my toots to reach other instances
```

Comment:
You might want to change this wording (and in the pt-BR translation too). To me this means my toots will ALWAYS reach other instances even if I set it to local-only. I suggest `Allow my toots to reach other instances by default`.

Reply:
Yeah, I ended up using the same wordind as the other `setting_default_*`, like the sensitive one that says "always mark media as sensitive". I do think your wording is better, I'll change it. Maybe the other one should be changed too :)